docs/Tutorials/3_Monarch_101.MD

```diff
@@ -0,0 +1,439 @@
+
+Now the key insight: **Forge services are ServiceActors that manage ActorMeshes of your ForgeActor replicas**.
+
+### The Service Creation Process
+
+```mermaid
+graph TD
+    subgraph ServiceCreation["Service Creation Process"]
+        Call["await PolicyActor.options(num_replicas=4, procs=1).as_service(model='Qwen')"]
+
+        ServiceActor["ServiceActor<br/>• Manages 4 replicas<br/>• Handles health checks<br/>• Routes service calls"]
+
+        subgraph Replicas["4 Independent Replicas"]
+            subgraph R0["Replica 0"]
+                PM0["ProcMesh<br/>1 process<br/>GPU 0"]
+                AM0["ActorMesh<br/>1 PolicyActor"]
+            end
+
+            subgraph R1["Replica 1"]
+                PM1["ProcMesh<br/>1 process<br/>GPU 1"]
+                AM1["ActorMesh<br/>1 PolicyActor"]
+            end
+
+            subgraph R2["Replica 2"]
+                PM2["ProcMesh<br/>1 process<br/>GPU 2"]
+                AM2["ActorMesh<br/>1 PolicyActor"]
+            end
+
+            subgraph R3["Replica 3"]
+                PM3["ProcMesh<br/>1 process<br/>GPU 3"]
+                AM3["ActorMesh<br/>1 PolicyActor"]
+            end
+        end
+
+        Call --> ServiceActor
+        ServiceActor --> R0
+        ServiceActor --> R1
+        ServiceActor --> R2
+        ServiceActor --> R3
+        PM0 --> AM0
+        PM1 --> AM1
+        PM2 --> AM2
+        PM3 --> AM3
+    end
+
+    style ServiceActor fill:#ffcc99
+    style AM0 fill:#99ff99
+    style AM1 fill:#99ff99
+    style AM2 fill:#99ff99
+    style AM3 fill:#99ff99
+```
+
+### Service Call to Actor Execution
+
+```mermaid
+graph TD
+    subgraph CallFlow["Complete Call Flow"]
+        UserCall["await policy_service.generate.route('What is 2+2?')"]
+
+        ServiceInterface["ServiceInterface<br/>• Receives .route() call<br/>• Routes to ServiceActor"]
+
+        ServiceActor["ServiceActor<br/>• Selects healthy replica<br/>• Load balancing logic<br/>• Failure handling"]
+
+        SelectedReplica["Selected Replica #2<br/>• ProcMesh with 1 process<br/>• ActorMesh with 1 PolicyActor"]
+
+        PolicyActor["PolicyActor Instance<br/>• Loads model<br/>• Runs vLLM inference<br/>• Returns 'The answer is 4'"]
+
+        GPU["GPU 2<br/>• vLLM engine<br/>• Model weights<br/>• KV cache<br/>• CUDA kernels"]
+
+        UserCall --> ServiceInterface
+        ServiceInterface --> ServiceActor
+        ServiceActor --> SelectedReplica
+        SelectedReplica --> PolicyActor
+        PolicyActor --> GPU
+
+        GPU -.->|"Response"| PolicyActor
+        PolicyActor -.->|"Response"| SelectedReplica
+        SelectedReplica -.->|"Response"| ServiceActor
+        ServiceActor -.->|"Response"| ServiceInterface
+        ServiceInterface -.->|"'The answer is 4'"| UserCall
+    end
+
+    style UserCall fill:#99ff99
+    style ServiceActor fill:#ffcc99
+    style PolicyActor fill:#cc99ff
+    style GPU fill:#ffcccc
+```
+
+## Multiple Services Sharing Infrastructure
```

Comment:
We are definitely shooting for this, but it sadly isn't there yet! You do have multiple services sharing now, but only in context of single node